### ISSUE TITLE
Need a guide for testing a provider using _run/kube example etc...

### ISSUE THREAD

**Issue by hydrogen18** [ovrclk/docs#164]
If someone is setting up a provider they shouldn't need to handle real AKT on mainnet just to test if their provider works.

Should add a no-frills guide to running the example from `_run/kube` etc. to run a local node & provider to verify the kubernetes cluster they want to sell on the network is ready for mainnet.

**Comment by boz** [comment]
Can you just point them to `_run/kube`?

**Comment by hydrogen18** [comment]
Yeah, but there are a few details that need to specified for them to get a successful deployment they can access. A technical user would figure it out anyways, but we might as well write it out for them to make it easier.
